extensions/toggl-track/src/index.tsx

```diff
@@ -0,0 +1,119 @@
+import dayjs from "dayjs";
+import duration from "dayjs/plugin/duration";
+import { AppContextProvider, useAppContext } from "./context";
+import RunningTimeEntry from "./components/RunningTimeEntry";
+import {
+  ActionPanel,
+  clearSearchBar,
+  Icon,
+  List,
+  PushAction,
+  showToast,
+  SubmitFormAction,
+  ToastStyle,
+} from "@raycast/api";
+import { TimeEntry } from "./toggl/types";
+import toggl from "./toggl";
+import { storage } from "./storage";
+import ProjectListItem from "./components/ProjectListItem";
+import CreateTimeEntryForm from "./components/CreateTimeEntryForm";
+
+dayjs.extend(duration);
+
+function ListView() {
+  const { isLoading, isValidToken, projectGroups, runningTimeEntry, timeEntries, projects } = useAppContext();
+  const getProjectById = (id: number) => projects.find((p) => p.id === id);
+
+  const timeEntriesWithUniqueProjectAndDescription = timeEntries.reduce((acc, timeEntry) => {
+    const existing = acc.find((t) => t.description === timeEntry.description && t.pid === timeEntry.pid);
+    if (!existing) {
+      acc.push(timeEntry);
+    }
+    return acc;
+  }, [] as TimeEntry[]);
+
+  async function resumeTimeEntry(timeEntry: TimeEntry) {
+    await toggl.createTimeEntry({
```

Comment:
```suggestion
    await showToast(ToastStyle.Animated, "Starting timer...");
    await toggl.createTimeEntry({
```
I do miss a loading toast, in multiple occasions I waited a few seconds for the success toast – I was not sure if I started it 😄 (Only added this one as example)